src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+public class RightTriangle extends Shape {
+    private double side;
+    private double base;
+
+    public RightTriangle(String color, double side, double base) {
+        super(Colors.valueOf(color));
```

Comment:
The `Colors.valueOf(color)` assumes the color string is always a valid enum constant. Consider adding error handling to manage cases where the color might not be valid.